rosdep/base.yaml

```
@@ -4974,6 +4974,10 @@ libpcsclite-dev:
   gentoo: [sys-apps/pcsc-lite]
   nixos: [pcsclite]
   ubuntu: [libpcsclite-dev]
+libpcsclite1:
+  debian: [libpcsclite1]
+  opensuse: [libpcsclite1]
+  ubuntu: [libpcsclite1]
```

I believe this is available in RHEL and Fedora as `pcsc-lite-libs`.  So:

```suggestion
  debian: [libpcsclite1]
  fedora: [pcsc-lite-libs]
  opensuse: [libpcsclite1]
  rhel: [pcsc-lite-libs]
  ubuntu: [libpcsclite1]
```